src/components/picker/PickerDialog.js

```diff
@@ -1,17 +1,15 @@
-import React from 'react';
+import React, {Component} from 'react';
 import {StyleSheet} from 'react-native';
 import PropTypes from 'prop-types';
 import _ from 'lodash';
 
-import {BaseComponent} from '../../commons';
-import {extractComponentProps} from '../../commons/modifiers';
 import Dialog from '../dialog';
 import View from '../view';
 import Text from '../text';
 import {Colors} from '../../style';
 
-class PickerDialog extends BaseComponent {
-  static displayName = 'IGNORE';
+class PickerDialog extends Component {
+  static displayName = 'NativePicker';
```

Comment:
You have 'NativePicker' already in the NativePicker component. This should be the 'PickerDialog' and the android one should be 'PickerDialog.android', no?

Reply:
Oops, bad copy-paste. 
And no, both should be `PickerDialog`, they can't co-exist. One is for iOS and the other is for Android